### PR TITLE
pAPRika v1.3.0

### PR DESCRIPTION
**Pull request by jaketanderson**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jaketanderson** [comment]
@jeff231li @slochower if either of you have a moment, could you review this PR to hopefully get the pAPRika version on conda-forge caught up with the latest bug fixes? Thanks.

File: .circleci/config.yml
Comment (jset231) [inline .circleci/config.yml:11]: I'm not too familiar with the specs of yaml files. Just to be sure, is this version correct (i.e. `ubuntu-2004:current`)?

File: .scripts/build_steps.sh
Comment (jset231) [inline .scripts/build_steps.sh:73]: I'm probably not up to date with the latest conda package. Is it correct to call it `conda-build`?

File: README.md
Comment (jset231) [inline README.md:88]: I'm guessing the `NumFOCUS` is on a new line because of linting, right?

**Review by jset231 — APPROVED** [review]
@jaketanderson Looks good to me. Awesome job!

File: README.md
Comment (jeff231li) [inline README.md:147]: I think it'll be a good idea for you to add your name as well here. =)

**Review by jeff231li — APPROVED** [review]
LGTM. (My mistake, I used a different account previously to approve the code)

File: README.md
Comment (slochower) [reply README.md:147]: Agree

File: LICENSE.txt
Comment (slochower) [inline LICENSE.txt:2]: SHould this be up to 2024?

File: .circleci/config.yml
Comment (jaketanderson) [reply .circleci/config.yml:11]: CircleCI has prebuilt images meant for testing; this line specifies use of the latest update of the Ubuntu 20.04 LTS prebuilt image. I think it's fine as is, but it may be best practice to use Ubuntu 24.04 LTS now, via `ubuntu-2404:current`.

File: README.md
Comment (jaketanderson) [reply README.md:88]: Right, this will be rendered as a badge in the README so the line spacing should not matter.

File: .scripts/build_steps.sh
Comment (jaketanderson) [reply .scripts/build_steps.sh:73]: It seems that `conda build` is actually an alias that calls the `conda-build` command. My guess is that `conda-smithy` changed this line to call the actual package instead of using an alias as a best practice.

File: LICENSE.txt
Comment (jaketanderson) [reply LICENSE.txt:2]: This is a copy of the license for conda-forge itself, and interestingly enough the latest public conda-forge license says "2015-2022". Here is the header on the copy that appears in the numpy feedstock, for example:

> BSD-3-Clause license
> Copyright (c) 2015-2022, conda-forge contributors
> All rights reserved.

https://github.com/conda-forge/numpy-feedstock/blob/main/LICENSE.txt

File: .circleci/config.yml
Comment (jaketanderson) [reply .circleci/config.yml:11]: On second thought, since numpy and scipy both use the conda-smithy-generated `ubuntu-2004:current` image, and it's the default given by rerendering conda-smithy, we should just use it too.

File: README.md
Comment (jaketanderson) [reply README.md:147]: The proper way of adding a person to the maintainer list is given here https://conda-forge.org/docs/maintainer/updating_pkgs/#updating-the-maintainer-list. Once this PR is merged I can start the process by creating an issue, then all that's needed is for one of the current maintainers to accept the PR that will be generated.

**Comment by jaketanderson** [comment]
@jeff231li and @slochower thank you for your comments. I think this PR is ready to be merged (if my comments sufficiently addressed yours), but only a current maintainer has the ability to do that.

File: .circleci/config.yml
Comment (jset231) [reply .circleci/config.yml:11]: Ah, that makes sense.

File: .scripts/build_steps.sh
Comment (jeff231li) [reply .scripts/build_steps.sh:73]: Nice, this looks okay to me.

File: README.md
Comment (jeff231li) [reply README.md:147]: Gotcha, it's been a while, and I forgot the process. But yes, after this PR let's get you on the list of the maintainers.

**Comment by jeff231li** [comment]
Looks good @jaketanderson. I'm approving and merging this.

**Pull request merged**
